src/evict/evict_lru.c

```diff
@@ -595,7 +595,8 @@ __evict_update_work(WT_SESSION_IMPL *session)
 	dirty_inuse = __wt_cache_dirty_leaf_inuse(cache);
 	if (__wt_eviction_dirty_needed(session, NULL))
 		F_SET(cache, WT_CACHE_EVICT_DIRTY | WT_CACHE_EVICT_DIRTY_HARD);
-	else if (dirty_inuse > (cache->eviction_dirty_target * bytes_max) / 100)
+	else if (dirty_inuse > (uint64_t)
```

Comment:
Not sure what coding conventions say about indentation here. @agorrod should take a final look at it.

Reply:
Review this indentation with Alex again

Reply:
Again the cast should be with the item. Break after the `>` if needed.